modules/op-providing-secure-connection.adoc

```diff
@@ -0,0 +1,41 @@
+[id='op-providing-secure-connection_{context}']
+= Providing secure connection with OpenShift Routes
+
+You can create a route with the re-encrypted TLS termination:
+
+[source,terminal,subs="atrributes+"]
+----
+oc create route reencrypt --service=<svc-name> --cert=tls.crt --key=tls.key --ca-cert=ca.crt --hostname=<hostname>
+----
+
+Alternatively, you can create a re-encrypted TLS termination YAML file to create a secured route.
+
+.Example Re-encrypt TLS Termination YAML of the Secured Route
```

Comment:
Use sentence case: Example re-encrypt TLS termination YAML to create a  secure route